test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -725,6 +725,32 @@ TEST_P(ExtAuthzHttpIntegrationTest, DefaultCaseSensitiveStringMatcher) {
   ASSERT_TRUE(header_entry.empty());
 }
 
+TEST_P(ExtAuthzHttpIntegrationTest, DeniedResponseWhenAuthzReturn5xx) {
```

Comment:
Also, we need to test the old behavior? (sending error, 403)

Reply:
sure. will add.